src/main/res/values/strings.xml

```diff
@@ -651,12 +651,12 @@
     <string name="options_close_all">Close All</string>
 
     <string name="pref_menus_optionsmenu_items_key" translatable="false">pref_menus_optionsmenu_items_3</string>
-    <string name="pref_menus_optionsmenu_items_title">Options menu items</string>
+    <string name="pref_menus_optionsmenu_items_title">Three-dot menu items</string>
```

Comment:
I did consider "Overflow menu" for the name, but it struck me as a bit of a misnomer since the items in that menu will always be there regardless of screen width. "Three-dot menu" seemed easiest to understand to me, but I can change it if you'd like.